section3/exercises/loops-arrays.js

```diff
@@ -18,21 +18,35 @@ for (var i = 0; i < animals.length; i++) {
 
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
+for ( var i = 0; i<animals.length; i++){
```

Comment:
Look up some `for loop` code examples on MDN docs! You have the functionality down and that's most important, but code style is _also_ important so other developers can read our code.